indigo_social/badges.py

```diff
@@ -259,22 +259,21 @@ class ContributorBadge(PermissionBadge):
     slug = 'contributor'
     name = 'Contributor'
     description = 'Can view work details'
-    permissions = ('indigo_api.view_work', 'indigo_api.add_annotation', 'indigo_api.change_annotation', 'indigo_api.delete_annotation')
+    permissions = ('indigo_api.view_work', 'indigo_api.add_annotation', 'indigo_api.change_annotation', 'indigo_api.delete_annotation', 'indigo_api.add_task')
 
 
 class DrafterBadge(PermissionBadge):
     slug = 'drafter'
     name = 'Drafter'
     description = 'Can create new works and edit the details of existing works'
-    permissions = ('indigo_api.add_work', 'indigo_api.change_work', 'indigo_api.add_document', 'indigo_api.change_document',
-                   'indigo_api.add_amendment', 'indigo_api.change_amendment', 'indigo_api.delete_amendment',)
```

Comment:
I think we should allow this to wrap at some point (eg. and group by model), otherwise it's going to get helluva long.